app/src/main/java/org/thoughtcrime/securesms/preferences/SettingsActivity.kt

```diff
@@ -498,27 +509,33 @@ class SettingsActivity : PassphraseRequiredActionBarActivity() {
 
             Cell {
                 Column {
+                    // add the debug menu in non release builds
+                    if (BuildConfig.BUILD_TYPE != "release") {
```

Comment:
Prevent string comparison and any potential typos?
```suggestion
                    if (BuildConfig.BUILD_TYPE == BuildConfig.DEBUG) {
```

Reply:
@AL-Session I considered this but I want to add a QA build type in the future. That build type should have the debug menu available, so unfortunately it needs to be compared to `release`.